tests/integration/migrations/utils.js

```diff
@@ -0,0 +1,276 @@
+var DB_PREFIX = 'medic_api_integration_tests__';
+
+var dbBackups;
+var _ = require('underscore'),
+    db = require('../../../api/db');
+
+function byId(a, b) {
+  if(a._id === b._id) {
+    return 0;
+  } else if(a._id < b._id) {
+    return -1;
+  } else {
+    return 1;
+  }
+}
+
+function matches(expected, actual) {
+  var i, k;
+
+  if(typeof expected === 'string') { return expected === actual; }
+  if(typeof expected === 'number') { return expected === actual; }
+  if(typeof expected === 'boolean') { return expected === actual; }
+  if(expected instanceof RegExp) {
+    return expected.test(actual);
+  }
+  if(Array.isArray(expected)) {
+    if(!Array.isArray(actual)) { return false; }
+    if(actual.length !== expected.length) { return false; }
+    for(i=expected.length-1; i>=0; --i) {
+      if(!matches(expected[i], actual[i])) {
+        return false;
+      }
+    }
+    return true;
+  } else {
+    if(!matches(Object.keys(expected).sort(), Object.keys(actual).sort())) { return false; }
+    for(k in expected) {
+      if(expected.hasOwnProperty(k)) {
+        if(!matches(expected[k], actual[k])) { return false; }
+      }
+    }
+    return true;
+  }
+}
+
+function assertDb(expectedContent) {
+  return new Promise(function(resolve, reject) {
+    db.request({
+      path: DB_PREFIX + 'medic/_all_docs',
+      method: 'GET',
+      qs: { include_docs: true },
+    }, function(err, result) {
+      if(err) {
+        return reject(err);
+      }
+
+      var actualContent = _.pluck(result.rows, 'doc').map(function(doc) {
+        return _.omit(doc, ['_rev']);
+      });
+
+      expectedContent.sort(byId);
+      actualContent.sort(byId);
+
+      // remove standard ddocs from actualContent
+      actualContent = actualContent.filter(function(doc) {
+        return doc._id !== '_design/medic' &&
+            doc._id !== '_design/medic-client';
```

Comment:
This is just removing ddocs...  Not sure if it would be useful to filter user docs as well, but I can see that might come up in the future.